presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -891,6 +913,34 @@ public void dropTable(String databaseName, String tableName, boolean deleteData)
         }
     }
 
+    private static void deleteTableFilesIfNeeded(HdfsContext context, HdfsEnvironment hdfsEnvironment, String databaseName, String tableName, Path tablePath)
+    {
+        try {
+            if (isS3Table(tablePath)) {
+                FileSystem fs = hdfsEnvironment.getFileSystem(context, tablePath);
+                if (fs.listFiles(tablePath, true).hasNext()) {
+                    log.warn(format("Table \"%s\".\"%s\" dropped, but files exist in '%s'", databaseName, tableName, tablePath.toUri()));
```

Comment:
Let's remove these log messages. We would only want to log a warning if the administrator could do something useful with this information.

Reply:
Since the deletion should be done by HMS and it might be fix sometime in the future,  I want to be able to track all cases that current situation happens. Would changing it to `log.info` makes sense for you ?